src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -284,10 +284,10 @@
             <% }) %>
             <% if (legend.length > 5) { %>
                 <div class="item expand">
-                    <a href="#" class="expand-legend">View <% print(legend.length - 5) %> more</a>
+                    <a href="#" class="expand-legend"><span class="i18n" data-i18n="View">View</span> <% print(legend.length - 5) %> <span class="i18n" data-i18n="more">more</span></a>
```

Comment:
Consider moving these "View More"/"View Less" elements across multiple lines to make them more readable now that they're nesting the `span`s.